auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/UserAuditService.kt

```diff
@@ -41,7 +41,7 @@ class UserAuditService(private val userAuditTrailRepo: UserAuditTrailRepo, priva
         )
     }
 
-    val userForgotPasswordAudit = insertSimpleAuditRowFun(UserAuditTrailRepo.AuditAction.FORGOT_PASSWORD_EMAIL)
+    val resetPasswordEmailAudit = insertSimpleAuditRowFun(UserAuditTrailRepo.AuditAction.RESET_PASSWORD_EMAIL)
```

Comment:
We should record the calling user in the `editing_user_cn` column of the table. leaving it null only if the user triggered it themselves.